public/css/dashboard.css

```diff
@@ -19,28 +23,30 @@ h3.pref-section-headline.remove {
 }
 
 h2.pref-headline.breach-summary {
-  margin-bottom: var(--padding);
+  margin-top: calc(var(--padding) * 5);
+  margin-bottom: 24px;
 }
 
 .pref {
   width: 100%;
-  max-width: 620px;
+  max-width: 600px;
```

Comment:
_Optional:_ Based on the utility class approach of this PR, could you include a `mw-600` class here instead of defining the attribute here?